docs/architecture/adr-028-public-key-addresses.md

```diff
@@ -135,7 +135,7 @@ type Addressable interface {
     Address() []byte
 }
 
-func NewComposed(typ string, subaccounts []Addressable) []byte {
+func Composed(typ string, subaccounts []Addressable) []byte {
```

Comment:
Why? I mean, I like the `New` naming pattern.

Not a big issue anyways....

Reply:
It's not a _constructor_. Also, we would need to make add `New` prefix to all functions in this package.